deepchem/feat/smiles_tokenizer.py

```diff
@@ -0,0 +1,183 @@
+# Requriments - transformers, tokenizers
+# Right now, the Smiles Tokenizer uses an exiesting vocab file from rxnfp that is fairly comprehensive and from the USPTO dataset.
+# The vocab may be expanded in the near future
+
+import collections
+import logging
+import os
+import re
+import numpy as np
+import pkg_resources
+from typing import List
+from transformers import BertTokenizer
+
+# export
+SMI_REGEX_PATTERN = r"(\[[^\]]+]|Br?|Cl?|N|O|S|P|F|I|b|c|n|o|s|p|\(|\)|\.|=|#|-|\+|\\|\/|:|~|@|\?|>>?|\*|\$|\%[0-9]{2}|[0-9])"
+
+def get_default_tokenizer():
```

Comment:
Could you add a docstring here?

Reply:
done!

Reply:
I might be missing it, but looks like `get_default_tokenizer()` doesn't have a docstring yet?

Reply:
woops confused docstring with pep8 code styling lol

Reply:
Looks like the docstring for `get_default_tokenizer()` still needs to be added here